css-brochure-form/styles.css

```diff
@@ -0,0 +1,85 @@
+h2,
+label,
+button {
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  text-transform: uppercase;
+}
+
+.input-text,
+#number-of-engineers {
+  font-family: 'AtlasGrotesk', sans-serif;
+  font-style: normal;
+}
+
+h2 {
+  color: rgba(250, 65, 65, 0.808);
+  font-size: medium;
+  letter-spacing: 1.5px;
+}
+
+h2,
+form {
+  text-align: center;
+}
+
+input {
+  width: 42%;
+  height: 2.5rem;
+  border-radius: 7px;
+  border: thin solid rgba(145, 145, 145, 0.377);
+  padding-left: 10px;
+  color: rgba(145, 145, 145, 0.377);
+  margin-bottom: 15px;
+}
+
+#label-email {
+  font-size: 8pt;
+  color: black;
+  font-weight: 500;
+  margin-right: 28em;
```

Comment:
That's a LOT of margin. What is the purpose of this? You can almost certainly accomplish whatever this is doing differently by setting a width for your form or something to that effect. Let me know if you have any questions on how to fix this.